lib/stub_ui/lib/src/ui/window.dart

```diff
@@ -35,6 +35,71 @@ typedef PlatformMessageResponseCallback = void Function(ByteData data);
 typedef PlatformMessageCallback = void Function(
     String name, ByteData data, PlatformMessageResponseCallback callback);
 
+/// Time-related performance metrics of a frame.
+class FrameTiming {
+  const FrameTiming._(this.buildStart, this.buildFinish, this.rasterStart, this.rasterFinish);
+
+  /// The time that UI thread starts building this frame.
+  ///
+  /// {@template dart.ui.FrameTiming.epoch}
+  /// This is a raw timestamp from some epoch. The epoch in all [FrameTiming] is
+  /// the same, but it may not match [DateTime]'s epoch.
+  /// {@endtemplate}
```

Comment:
this comment makes no sense, since it's a `Duration`.

Reply:
or maybe the comment makes sense but the use of `Duration` doesn't? This API confuses me.

Reply:
Maybe these should just be `int`s?